minetest.conf.example

```diff
@@ -149,6 +149,12 @@
 #    Values larger than 26 will start to produce sharp cutoffs at cloud area corners.
 #cloud_radius = 12
 #enable_3d_clouds = true
+#    Enable random menu image, the subgame must support this by having multiple
+#    images in the menu folder, named "background (or overlay).png.x" where x
+#    is a number, increasing from 1. The subgame also needs to put
+#    "number_of_images = x" into game.conf, where x is the amount of images
+#    with a number after them.
+#random_menu_images = false
```

Comment:
random_menu_images is there so that the player can use it without editing game.conf. The subgame maker would add the number_of_images setting to game.conf if s/he wanted players to be able to use it. At least, that was my reasoning!

Thanks for your comments and help!